internal/upgrades/upgradevalidation/validation_test.go

```diff
@@ -129,30 +129,6 @@ func (s *upgradeValidationSuite) TestCheckForDeprecatedUbuntuSeriesForModel(c *g
 	c.Assert(blocker.Error(), gc.Equals, `the model hosts 1 ubuntu machine(s) with an unsupported base. The supported bases are: ubuntu@24.04, ubuntu@22.04, ubuntu@20.04`)
 }
 
-func (s *upgradeValidationSuite) TestGetCheckUpgradeSeriesLockForModel(c *gc.C) {
```

Comment:
More references in this file to remove:

- 38:		*upgradevalidation.NewBlocker("unexpected upgrade series lock found"),
- 43:			*upgradevalidation.NewBlocker("unexpected upgrade series lock found"),
- 51:- unexpected upgrade series lock found
- 53:- unexpected upgrade series lock found
- 56:- unexpected upgrade series lock found
- 59:- unexpected upgrade series lock found
- 62:- unexpected upgrade series lock found
- 103:			return upgradevalidation.NewBlocker("unexpected upgrade series lock found"), nil
- 112:- unexpected upgrade series lock found`[1:])